docs/antora/modules/command-line/pages/command_line_tools.adoc

```diff
@@ -0,0 +1,376 @@
+= Command Line Tools
+keywords: queue, java, command line,
+author: Kevin Powe
+:reftext: Command Line Tools
+:navtitle: Comamnd Line Tools
+:source-highlighter: highlight.js
+
+== Overview
+
+The following command line tools can be used to interact with Chronicle Queue.
+
+[cols="1,1"]
+|===
+
+| ★ <<_archiverollfiles,*ArchiveRollFiles*>>
+| Assists in management of roll cycle files
+
+| <<_chroniclehistoryreadermain,*ChronicleHistoryReaderMain*>>
+| Reads history from queue and outputs latency histograms
+
+| <<_chroniclereadermain,*ChronicleReaderMain*>>
+| Displays records from a queue in text form, supporting more complex operations
+
+| <<_chroniclewritermain,*ChronicleWriterMain*>>
+| Write contents of files provided to queue
+
+| <<_dumpmain,*DumpMain*>>
+| Performs simple text output of queue contents
+
+| <<_refreshmain,RefreshMain>>
+| Refreshes the directory listing for a queue
+
+| <<_unlockmain,*UnlockMain*>>
+| Manually force-unlocks a queue
+|===
+
+=== Using Command Line Tools
+
+Where noted, each command line tool can be invoked via a shell script. Command line tools can also be invoked using Maven, assuming a project POM file that includes the Chronicle Queue artifact. For example:
+
+[source, shell script]
+----
+$ mvn exec:java -Dexec.mainClass="net.openhft.chronicle.queue.main.DumpMain" -Dexec.args="myqueue"
+----
+
+In the above command invoking *DumpMain*,  _myqueue_ is the directory containing your .cq4 files.
+
+You can also set up any dependent files manually. This requires the `chronicle-queue.jar`, from any version 4.5.3 or later, and that all dependent files are present on the class path. The dependent jars are listed below:
+
+[source, shell script]
+----
```

Comment:
I recommend removing this as the versions are out of date so might be confusing

Reply:
Done - thank you for the feedback!